src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4346,7 +4362,26 @@ public override bool PreVisit(CallNode node)
                 _currentScope = scopeNew.Parent;
                 PostVisit(node.Args);
 
-                // Typecheck the invocation.
+                if (maybeFunc.HasIdentifiers && _features.HasFlag(Features.SupportColumnNamesAsIdentifiers))
+                {
+                    var i = 0;
+
+                    foreach (var arg in args)
+                    {
+                        i++;
```

Comment:
>i++; [](http://example.com/codeflow?start=24&length=4)

This means arg is args[i-1] . 
so we're out of sync with IsIdentifierParam(i)?

Reply:
line 4367 should start at -1 instead

Reply:
This i++ is wrongly placed, you're right
It should be after the 'if' condition